QUEST_LV_0100.tsv

```diff
@@ -4871,74 +4871,74 @@ QUEST_LV_0100_20150317_004870	Hand over Big Cockatrice tails to Tara Miles
-QUEST_LV_0100_20150317_004900	You were able to charge the absorbing stones with monsters' vitalities. Go back to Grita.
-QUEST_LV_0100_20150317_004901	{memo X}You finished charging monsters' vitalities to the absorbing stones. Talk to Grita about the method of activating the barrier device.
-QUEST_LV_0100_20150317_004902	{memo X}Activate the barrier device
-QUEST_LV_0100_20150317_004903	{memo X}The barrier device is at the end of the left hallway. Activate the barrier device.
-QUEST_LV_0100_20150317_004904	{memo X}You were able to activate the barrier device. Talk to Grita again.
-QUEST_LV_0100_20150317_004905	You arrived on the 2nd floor of the Mage Tower. Talk with Grita what to do next.
-QUEST_LV_0100_20150317_004906	Search for the Jewel of Prominence
-QUEST_LV_0100_20150317_004907	Find the Jewel of Prominence. Use the search stick to look for the Jewel of Prominence on the 2nd floor of the Mage Tower.
-QUEST_LV_0100_20150317_004908	You found the Jewel of Prominence. Talk to Grita.
-QUEST_LV_0100_20150317_004909	You got the Jewel of Prominence. Talk with Grita about how you can input the power into the Jewel of Prominence.
-QUEST_LV_0100_20150317_004910	Collect the Steam of Flame
-QUEST_LV_0100_20150317_004911	Grita told you that in order to complete the Jewel of Prominence, you should fill the Steam of Flame into it. Fill it with the Steam of Flame that leaks from the big reading room.
-QUEST_LV_0100_20150317_004912	You filled the Jewel of Prominence with the Steam of Flame. Go back to Grita.
-QUEST_LV_0100_20150317_004913	You collected all the Steam of Flame. Ask Grita wha tto do next to complete the Jewel of Prominence.
-QUEST_LV_0100_20150317_004914	Charge the jewel
-QUEST_LV_0100_20150317_004915	{memo X}If you unleash the Steam of Flame, it will bring the Essences of Flame. Fill the Essences to the 2nd pot.
-QUEST_LV_0100_20150317_004916	You filled the Essences into the Jewel of Prominence in full. Talk to Grita.
-QUEST_LV_0100_20150317_004917	You filled all the Essences. Ask Grita what to do next.
-QUEST_LV_0100_20150317_004918	Obtain the Sources of Flame by defeating monsters
-QUEST_LV_0100_20150317_004919	{memo X}The energy is not enough yet, so Grita told you that you should enhance the energy with the Sources of Flame. Defeat the monsters nearby and collect the Sources of Flame.
-QUEST_LV_0100_20150317_004920	You collected enough Sources of Flame. Talk with Grita again.
+QUEST_LV_0100_20150317_004884	$You successfully lit up the signal. Talk to Grita again.
+QUEST_LV_0100_20150317_004885	$Defeat the Rubblems
+QUEST_LV_0100_20150317_004886	$You lit up a signal to let the Goddess know that help is coming. Discuss with Grita on what to do next.
+QUEST_LV_0100_20150317_004887	$Check the 1st Transport Magic Field
+QUEST_LV_0100_20150317_004888	$Grita told you that using the Transport Magic Field is the fastest way to go upstairs. Go to the 1st Transport Magic Field to see whether it is activating alright.
```

Comment:
"activating alright" -> "working correctly" ?
or "see if it activates properly" ?